### PR TITLE
Feature/log

### PR DESCRIPTION
**Pull request by lats1**

### CODE REVIEW

File: Controller/ProcessController.php [hunk 1]
```diff
@@ -150,8 +152,12 @@ public function new(Request $request, ProcessManager $processManager): Response
     /**
      * @Route("/{id}", name="process_show", methods={"GET", "POST"})
      */
-    public function show(Request $request, Process $process): Response
+    public function show(Request $request, Process $process, LoggerInterface $logger): Response
```
Comment (tuj) [inline Controller/ProcessController.php]: What do you do with the LoggerInterface?

File: EventListener/LoggableListener.php [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+        if (!empty($logEntry)) {
+            if ($object instanceof Process) {
+                $this->createProcessLogEntry(
+                    $logEntry,
+                    $object,
+                    $this->getLevel($action),
+                    $ea
+                );
+            //} else if (method_exists($object, 'getProcess')) {
```
Comment (tuj) [inline EventListener/LoggableListener.php]: Remove commented code

File: Repository/ProcessLogEntryRepository.php [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    public function __construct(RegistryInterface $registry)
+    {
+        parent::__construct($registry, ProcessLogEntry::class);
+    }
+
+    // /**
+    //  * @return ProcessLogEntry[] Returns an array of ProcessLogEntry objects
+    //  */
+    /*
+    public function findByExampleField($value)
```
Comment (tuj) [inline Repository/ProcessLogEntryRepository.php]: Remove

File: Controller/ProcessController.php (hunk 1 above)
Comment (lats1) [reply Controller/ProcessController.php]: Nothing :) Fixed

File: EventListener/LoggableListener.php (hunk 2 above)
Comment (lats1) [reply EventListener/LoggableListener.php]: Fixed in commit #6580aeb

File: Repository/ProcessLogEntryRepository.php (hunk 3 above)
Comment (lats1) [reply Repository/ProcessLogEntryRepository.php]: Fixed in commit #ebdc30a

**Pull request merged**